README.md

```diff
@@ -87,8 +90,8 @@ upstream and may support more recent version of AOTriton
 |-----------------------|-------------------------------------------------|
 |  2.2 and earlier      |               N/A, no support                   |
 |        2.3            |                   0.4b                          |
-|        2.4            |                   0.7b (backported)             |
-|        2.5            |                   0.8b (backported)             |
+|        2.4            |                   0.9b (backported)             |
+|        2.5            |                   0.9b (backported)             |
 |        2.6            |                   0.9b (backported)             |
 |        2.7            |                   0.9b (backported)             |
 |        2.8            |                   0.10b (once released)         |
```

Comment:
Why is it that we don't list the '0.10b drop-in only" clauses here for any branch that supports 0.9b? Wouldn't the same logic apply as for upstream branches?

Reply:
The same logic applies, but backport happens very frequently on ROCm forks and they will be moved to latest version at some point...